packages/bits/package.json

```diff
@@ -89,25 +89,17 @@
   },
   "analyze": false,
   "peerDependencies": {
-    "@angular-devkit/schematics": "^11.1.4",
```

Comment:
I haven't seen this library in any of the angular libraries' peerDependencies. Schematics work without it.